services/apps/src/apps/routes/admin.clj

```diff
@@ -153,6 +156,58 @@
           on the fields included in the request."
           (ok (apps/admin-update-category current-user (assoc body :id category-id)))))
 
+(defroutes* admin-ontologies
+
+  (GET* "/" []
+        :query [params SecuredQueryParams]
+        :return ActiveOntologyDetailsList
+        :summary "List Ontology Details"
+        :description
+"Lists Ontology details saved in the metadata service.
+
+#### Delegates to metadata service
+    GET /ontologies"
```

Comment:
Another "might be nice" sort of comment -- in data-info we have `get-error-code-block` which we use for automatically and consistently building the "Error Codes:" blocks in the swagger docs. It might be nice to create a similar helper function for delegations like this.